src/components/Tab/Tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: Tab;
+  onTableSelected: (tabId: string) => void;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, onTableSelected, selectedTabId } = props;
+
+  return (
+
+    <>
+      <div className="card-link">
+
+        {tabs.map((tab) => {
+          return (
+            <div key={tab.id}>
+              <button
+                type="submit"
+                className="btn btn-link"
+                onClick={() => onTableSelected(tab.id)}
```

Comment:
![image](https://user-images.githubusercontent.com/64437769/133753129-2a0cbd8e-6539-446f-a8b9-acd5ac52ca38.png)

Reply:
here nothing changed. You still call the callback every time pressing on tabs